lib/galaxy/webapps/galaxy/api/workflows.py

```diff
@@ -74,6 +76,11 @@ def index(self, trans, **kwd):
             encoded_id = trans.security.encode_id(wf_sa.stored_workflow.id)
             item['url'] = url_for( 'workflow', id=encoded_id )
             item['owner'] = wf_sa.stored_workflow.user.username
+            item['latest_workflow_steps'] = len( wf_sa.stored_workflow.latest_workflow.steps )
+            item['user_email'] = wf_sa.stored_workflow.user.email
```

Comment:
I think this line should be removed, it's not necessary to expose the user email through the API, the owner should be enough for what you are doing.